lib/dash/mpd_utils.js

```diff
@@ -545,6 +545,8 @@ shaka.dash.MpdUtils = class {
  *   end: number
  * }}
  *
+ * @export
```

Comment:
Why? I think that it's not necessary

Reply:
Hi! this needs to be there since TimelineSegmentIndex is exported to be unit tested. Since the TimelineSegmentIndex uses `SegmentTemplateInfo` in its constructor (which uses TimeRange), TimeRange needs to be exported. If you've got an idea for a way around this I'm open to suggestions!

Reply:
@joeyparrish can you help here?

Reply:
@joeyparrish friendly reminder!

Reply:
Unit tests run on uncompiled code, so no export is necessary.  Only full-stack integration tests that explicitly load a compiled bundle use the compiled code, and those tests don't isolate internal components.  So you shouldn't need to export for testing.